golang/internal/config/config.go

```diff
@@ -37,6 +37,7 @@ type CommonConfiguration struct {
 	ImportContainerImage     string        `yaml:"importContainerImage"     env:"IMPORT_CONTAINER_IMAGE"      env-default:"rclone/rclone:1.57.0"` //nolint:lll
 	RootDomain               string        `yaml:"rootDomain"               env:"ROOT_DOMAIN"                 env-default:""`
 	ReadHeaderTimeout        time.Duration `yaml:"readHeaderTimeout"        env:"READ_HEADER_TIMEOUT"         env-default:"15s"`
+	HostAddress              string        `yaml:"hostAddress"              env:"HOST_ADDRESS"                env-default:""`
```

Comment:
Could we add a comment in here that we are using this in darklens only?

Reply:
We could also use it in dyo, but sure :)